source/index.html

```diff
@@ -1,12 +1,173 @@
+            </p>
+            <div class="advantage__image">
+              <img src="img/mobile/index/advantage-1.jpg"  alt="Достопримечательность Седоны" width="320" height="260">
+            </div>
+            <ul class="reasons__list">
+              <li class="reason_item">
+                <h4 class="reason-title">Жилье</h4>
+                <p class="reaseon__text">
+                  Рекомендуем жить в настоящем мотеле, все как в кино!
+                </p>
+              </li>
+              <li class="reason_item">
+                <h4 class="reason-title">Сувениры</h4>
+                <p class="reaseon__text">
+                  Не только китайского, но и местного производства!
+                </p>
+              </li>
+              <li class="reason_item">
+                <h4 class="reason-title">Еда</h4>
+                <p class="reaseon__text">
+                  Всегда заказывайте фирменный бургер, вы не разочаруетесь!
+                </p>
+              </li>
+            </ul>
+          </li>
+          <li class="advantages__item advantages__item--color">
+            <h3 class="advantage__title">Там есть мост дьявола</h3>
+            <div class="advantage__number">- №2 -</div>
+            <p class="advantage__text">
+              Да, по нему можно пройти! Если вы осмелитесь, конечно
+            </p>
+            <div class="advantage__image">
+              <img src="img/mobile/index/advantage-2.jpg"  alt="Достопримечательность Седоны" width="320" height="260">
+            </div>
+          </li>
+          <li class="advantages__item advantages__item--color">
+            <h3 class="advantage__title">Небольшая площадь</h3>
+            <div class="advantage__number">- №3 -</div>
+            <p class="advantage__text">
+              Все интересные места находятся очень близко
+            </p>
+          </li>
+          <li class="advantages__item advantages__item--color">
+            <h3 class="advantage__title">Красивая дорога</h3>
+            <div class="advantage__number">- №4 -</div>
+            <p class="advantage__text">
+              Ехать в седону из Лас-Вегаса совсем не скучно!
+            </p>
+          </li>
+          <li class="advantages__item advantages__item--color">
+            <h3 class="advantage__title">Мало туристов</h3>
+            <div class="advantage__number">- №5 -</div>
+            <p class="advantage__text">
+              Большинство едет в Гранд-Каньон и толпится там
+            </p>
+          </li>
+        </ol>
+      </div>
+    </section>
+    <section class="booking">
+      <div class="booking__wrapper">
+        <h2 class="visually-hidden">Поиск гостиницы</h2>
+        <p class="booking__text booking__text--uppercase"><b>Заинтересовались?</b></p>
+        <p class="booking__text">Укажите предполагаемые даты поездки, и мы покажем Вам лучшие предложения гостиниц в Седоне</p>
+        <a class="booking__button button" href="#">Найти гостиницу</a>
```

Comment:
чтобы не перекидывало на вверх страницы при клике, можно везде знак восклицания указывать, где заглушка стоит
```suggestion
        <a class="booking__button button" href="#!">Найти гостиницу</a>
```